docs/pages/enroll-resources/workload-identity/aws-oidc-federation.mdx

```diff
@@ -0,0 +1,431 @@
+</TabItem>
+</Tabs>
+
+### Configuring RBAC
+
+First, you'll create an IAM role. This role will be assumed by the workload
+after it authenticates to AWS using the JWT SVID issued by Teleport Workload
+Identity.
+
+When creating the IAM role, you'll define a trust policy that controls which
+workload identities are able to assume the role. This policy will contain
+conditions which will be evaluated against the claims within the JWT SVID
+issued by Teleport Workload Identity. In our case, the only claim we want to 
+evaluate is the `sub` claim, which will contain our workload's SPIFFE ID.
+
+With the IAM role created, you will then attach an IAM policy to it to define
+what privileges the workload will have when it assumes the role.
+
+<Tabs>
+<TabItem label="Terraform">
+
+```hcl
+// Create a role that the workload identity will assume.
+resource "aws_iam_role" "example" {
+  // Choose a unique, meaningful name that describes the role and the workload
+  // that will assume it.
+  name = "workload-id-demo"
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [{
+      Effect = "Allow"
+      Principal = {
+        Federated = aws_iam_openid_connect_provider.example.arn
+      }
+      Action = "sts:AssumeRoleWithWebIdentity"
+      Condition = {
+        StringEquals = {
+          "${aws_iam_openid_connect_provider.example.url}:aud" = "sts.amazonaws.com"
+          "${aws_iam_openid_connect_provider.example.url}:sub" = "spiffe://example.teleport.sh/svc/example-service"
+        }
+      }
+    }]
+  })
+}
+
+resource "aws_iam_policy" "example" {
+  // Choose a unique, meaningful name that describes what the policy grants
+  // access to.
+  name        = "workload-id-s3-full-access"
+  path        = "/"
+
+  // This example policy grants full access to AWS S3. In production, you
+  // may wish to grant a less permissive policy.
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [{
+      Action = "s3:*"
+      Effect = "Allow"
+      Resource = [
+        aws_s3_bucket.workload_id_demo.arn,
+        "${aws_s3_bucket.workload_id_demo.arn}/*"
+      ]
+    }]
+  })
+}
+
+// Attach the policy to the role
+resource "aws_iam_role_policy_attachment" "example" {
+    role       = aws_iam_role.example.name
+    policy_arn = aws_iam_policy.example.arn
+}
+```
+
+</TabItem>
+<TabItem label="AWS Console">
+
+### Creating an IAM Policy
+
+1. Browse to IAM
+2. Select "Policies" from the sidebar
+3. Click "Create policy"
+4. Choose the "S3" service
+5. Under "Actions allowed", choose "All S3 actions"
+6. Under "Resources", choose "Specific"
+  6a. For "bucket" enter the name of the bucket you created earlier.
+  6b. For "object" enter the name of the bucket you created earlier and select
+    "All objects"
+7. Click "Next"
+8. Enter a unique and meaningful name for this policy, in our example, this will
+  be `workload-id-s3-full-access`
+9. Click "Create policy"
+
+### Creating an IAM Role
+
+1. Browse to IAM
```

Comment:
All numbered list items begin with `1.` (a `remark` linter enforces this, so you should see a lint error)